src/modules/entities/entities-selectors.js

```diff
@@ -1,20 +1,57 @@
 import { createSelector } from "reselect";
-import { denormalize } from "normalizr";
 
-import { users } from "modules/entities/entities-schema";
+const getEntities = state => state.entities;
 
-const getUserIds = state => state.user.users;
+const getSkills = createSelector(
+  getEntities,
+  entities => entities.skills
+);
 
-const getEntities = state => state.entities;
+const getNormalizedUsers = createSelector(
+  getEntities,
+  entities => entities.users
+);
 
-export const getUsers = createSelector(
+const getNormalizedUserSkills = createSelector(
   getEntities,
-  getUserIds,
-  (entities, ids) => {
-    const r = denormalize(ids, users, entities);
+  entities => entities.userSkills
+);
+
+const getUserSkills = createSelector(
```

Comment:
This should return an object with the same shape as `getNormalizedUserSkills` but its values will have a skill object inside instead of the id.

If `getNormalizedUserSkills` returns:
```ts
{
  'user-1-skill-1': {
    level: 2,
    skill: 'skill-1'
  },
  'user-2-skill-2': {
    level: 5,
    skill: 'skill-2'
  }
}
```

then `getUserSkills` should return:
```ts
{
  'user-1-skill-1': {
    level: 2,
    skill: {
      id: 'skill-1',
      name: 'foo'
    }
  },
  'user-2-skill-2': {
    level: 5,
    skill: {
      id: 'skill-2',
      name: 'bar'
    }
  }
}
```